packages/react-components/src/lib/cesium-map/tools/draw/drawHelper.ts

```diff
@@ -1012,6 +1015,20 @@ var DrawHelper = (function () {
     ]);
   }
 
+  // ALEX add to change draw mode cursor
+  function setDrawCursor(scene) {
+    if (scene) {
+      scene.canvas.style.cursor = 'crosshair';
```

Comment:
can the draw cursor be configured by the user?

Reply:
As you see currently not, but if there is a need we can arrange it